scheduled/redshift.js

```diff
@@ -8,7 +8,7 @@ const {Client} = require('pg')
 const copyToSteam = require('pg-copy-streams').to
 const rollbar = require('../config/rollbar')
 const AWS = require('aws-sdk')
-const dateUtil = require('./util/date-util')
+const dateUtil = require('../db/util/date-util')
```

Comment:
This can move along with the `redshift.js` since it's the only file that uses it.